Day26/Ex5.sql

```diff
@@ -0,0 +1,7 @@
+
+Use shoppingdb;
+
+update product as p
+set p.SalePrice = p.SalePrice * (1 + 10/100)
+where p.SalePrice <= 250000
```

Comment:
Not wrong, but should be consistent in tying. In other places you use uppercase - "UPDATE", "SET" and "WHERE", but here you use lowercase - "update", "set" and "where".